src/components/ImageWrap.js

```diff
@@ -36,8 +36,7 @@ class ImageWrap extends React.Component {
           paddingTop: paddingRef[aspectRatio],
           opacity: showImage ? 1 : 0,
           top: showImage || skipIntro ? '0px' : '15px',
-          transition: 'opacity 0.5s, top 0.5s',
-          transitionDelay: '1s'
+          transition: 'all .3s ease-in-out'
```

Comment:
I have an edit for this so that we don't lose the loading transitions.
This should work. The shorthand represents 'transition-property transition-duration easing-function transition-delay'
`transition: opacity 0.5s ease 1s, top 0.5s ease 1s, transform 0.3s ease-in-out 0s`